kong/db/migrations/core/005_120_to_130.lua

```diff
@@ -55,9 +68,10 @@ return {
         PRIMARY KEY (partition, id)
       );
 
-
-
       ALTER TABLE routes ADD headers map<text,frozen<set<text>>>;
+      ALTER TABLE services ADD client_certificate_id uuid;
+
+      CREATE INDEX IF NOT EXISTS services_client_certificate_id_idx ON services(client_certificate_id);
```

Comment:
style: in terms of migrations styling, we follow a rule to have 3 line breaks between statements related to a given feature (hence why the line breaks you removed here. We apply this for both Postgres and Cassandra migrations.